health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SearchChecklist.js

```
@@ -129,8 +129,9 @@ const SearchChecklist = () => {
           onStepClick={onStepClick}
           activeSteps={6}
         />
+        <div style={{height:"2rem"}}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)